redi-connect-front/src/components/organisms/PreFooter.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import {
+  Container,
+  Section,
+  Columns,
+  Heading,
+  Content,
+  Image,
+} from "react-bulma-components";
+import Button from "../atoms/Button";
+import hello from "../../assets/images/hello.svg";
+// import helloMobile from "../../assets/images/hello-mobile.svg";
+import "./PreFooter.scss";
+
+const PreFooter = () => (
+  <Container className="pre-footer-container">
```

Comment:
I think the ideo of the bulma is to put the container inside of the section. 
I also did it like this and have seen that it does not work good with the spacing. 
Also I've seen you are adding spacing to the container, I think the spacing was originally planned between sections.